deployment/.env

```diff
@@ -1,2 +1,10 @@
-DOCKER_TAG=latest
 REPOSITORY_DOCKER_URL=ghcr.io/nasa-ammos
+DOCKER_TAG=latest
+
+# Provide Usernames and Passwords Below
+AERIE_USERNAME=
+AERIE_PASSWORD=
+
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
```

Comment:
Sorry, I'm kind of confused about this file. It's checked into git, yet these fields have to be modified per-user? In that case, shouldn't this not be checked in?

Reply:
Or maybe we should change this to `deployment/.env.template` and suggest users use a similar pattern as the top-level `.env.template`.

Reply:
We need it because it's the default .env file for the deployment. It comes with the deployment .zip as an example of what env vars are needed to deploy.

Reply:
Ohh I see! Thanks again, this makes sense!